site/content/en/reference/live/install-resource-group.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Install-resource-group"
+linkTitle: "install-resource-group"
+type: docs
+description: >
+   Apply ResourceGroup custom resource definition to the cluster
+---
+<!--mdtogo:Short
+    Apply ResourceGroup custom resource definition to the cluster
+-->
+
+**NOTE**: This command is not available unless the `RESOURCE_GROUP_INVENTORY`
```

Comment:
Worth repeating here and for migrate command that this is only relevant for the alpha feature. Actually use the name of the relevant feature to tie these pages together.

Reply:
I've updated this initial **NOTE** to include the following: 
```
This is an alpha command for the new ResourceGroup as inventory object
 functionality
```

Please let me know if this is what you are looking for.